app/elements/pw-shell.html

```diff
@@ -164,11 +173,17 @@
 
         <nav>
           <iron-selector class="sections-tabs" attr-for-selected="name" activate-event="" selected="[[_selectedSection(_sectionData.*)]]">
-            <a href="/2.0/start/" name="1.0/start">Start</a>
-            <a href="/2.0/docs/devguide/feature-overview" name="2.0/docs">Polymer</a>
-            <a href="/2.0/toolbox/" name="1.0/toolbox">App Toolbox</a>
-            <a href="/2.0/blog/" name="1.0/blog">News</a>
-            <a href="https://elements.polymer-project.org/" target="_blank" name="catalog">Element Catalog</a>
+
+            <a href="/1.0/start/" name="1.0/start" hidden="[[_shouldShowV2(activeDocsVersion)]]">Start</a>
+            <a href="/1.0/docs/devguide/feature-overview" name="1.0/docs" hidden="[[_shouldShowV2(activeDocsVersion)]]">Polymer</a>
+            <a href="/1.0/toolbox/" name="1.0/toolbox" hidden="[[_shouldShowV2(activeDocsVersion)]]">App Toolbox</a>
+
+            <a href="/2.0/start/" name="2.0/start" hidden="[[!_shouldShowV2(activeDocsVersion)]]">Start</a>
+            <a href="/2.0/docs/about_20" name="2.0/docs" hidden="[[!_shouldShowV2(activeDocsVersion)]]">Polymer</a>
+            <a href="/2.0/toolbox/" name="2.0/toolbox" hidden="[[!_shouldShowV2(activeDocsVersion)]]">App Toolbox</a>
```

Comment:
My only other suggestion is to style away the irrelevant links here on the `active-docs-version` attribute as well (similar to the mobile drawer) to avoid the work of `_shouldShowV2`:

```css
:host([active-docs-version="1.0"]) .section-tabs a[data-version="2.0"] { ... }
```